stacks/secrets.ts

```diff
@@ -1,14 +1,21 @@
 import { StackContext } from '@serverless-stack/resources';
 import { Secret } from 'aws-cdk-lib/aws-secretsmanager';
 
+const LINKEDIN_CLIENT_ID = process.env.LINKEDIN_CLIENT_ID;
+const LINKEDIN_CLIENT_SECRET = process.env.LINKEDIN_CLIENT_SECRET;
+
 export function Secrets({ stack, app }: StackContext) {
+  if (!LINKEDIN_CLIENT_ID || !LINKEDIN_CLIENT_SECRET) {
```

Comment:
The issue is that this is not just required at stack creation but every time. Now everyone who develops it should have those env vars set? I'm not a big fan of that

What I want to do is use https://docs.sst.dev/constructs/Secret - but we can't access those secrets in CDK unfortunately. I made an issue for it: https://github.com/serverless-stack/sst/issues/1984 maybe give it a 👍🏻